# Device information message

# Node address
NodeAddr node_addr

# Procedure status
ProcStatus[] proc_status_vec

# Is procedure status expired
bool is_proc_status_expired
